webapi/src/test/scala/org/knora/webapi/models/standoffmodels/StandoffModels.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.models.standoffmodels
+
+import org.knora.webapi.feature.FeatureFactoryConfig
+import org.knora.webapi.messages.{OntologyConstants, StringFormatter}
+import org.knora.webapi.messages.util.rdf.JsonLDKeywords
+import org.knora.webapi.messages.v2.responder.standoffmessages.{
+  CreateMappingRequestMetadataV2,
+  CreateMappingRequestV2,
+  CreateMappingRequestXMLV2
+}
+import org.knora.webapi.sharedtestdata.SharedTestDataV1.ANYTHING_PROJECT_IRI
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+import org.knora.webapi.messages.IriConversions._
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+
+import java.util.UUID
+
+sealed abstract case class DefineStandoffMapping private (
```

Comment:
docstrings are missing for the case class and all its methods. Is there a reason why?